src/makeInfinityAdder.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let total = 0;
+
+  const sum = (num) => {
+      if (typeof num !== "number") {
+        const result = total;
+
+        total = 0;
+
+        return result;
+    }
+    total += num;
```

Comment:
Are you using autoformat? If yes - turn it off.

1. 2-space indentations
2. Singlequotes for strings

Reply:
Just disabled PRETTIER. :(